content/applications/general/export_import_data.rst

```diff
@@ -350,62 +418,63 @@ It will produce the following CSV file:
-imported these two CSV files, you will have 4 contacts and 3 companies. (the firsts two contacts are
-linked to the first company). You must first import the companies and then the persons.
+imported these two :abbr:`CSV (Comma-separated Values)` files, there are four contacts and three
+companies (the first two contacts are linked to the first company). Keep in mind to first import
+the companies and then the persons.
 
 How to adapt an import template
 ===============================
 
-Import templates are provided in the import tool of the most common data to
-import (contacts, products, bank statements, etc.).
-You can open them with any spreadsheets software (Microsoft Office,
+Import templates are provided in the import tool of the most common data to import (contacts,
+products, bank statements, etc.). Open them with any spreadsheet software (Microsoft Office,
 OpenOffice, Google Drive, etc.).
 
-How to customize the file
-=========================
+How to customize file
+=====================
 
-* Remove columns you don't need. We advise to not remove the *ID* one (see
-  why here below).
+* Remove columns that are not needed. Odoo advises not removing the :guilabel:`External ID` (ID)
+  column (see why below).
 * Set a unique ID to every single record by dragging down the ID sequencing.
 
   .. image:: export_import_data/dragdown.gif
      :align: center
+     :alt: Animation of a spreadsheet application automatically populating a column in the ID.
 
-* When you add a new column, Odoo might not be able to map it automatically if its
-  label doesn't fit any field of the system.
-  If so, find the corresponding field using the search.
+* When adding a new column, Odoo might not map it automatically if its label does not match any
+  field in the system. If so, find the corresponding field using the search.
 
     .. image:: export_import_data/field_list.png
        :align: center
+       :alt: Searching for the field in the drop-down menu of the import page.
 
-  Then, use the label you found in your import template in order to make it work
-  straight away the very next time you try to import.
+  Then, use the label found in the import template to make it work right away the next time the
+  import is attempted.
 
 Why an “ID” column
 ==================
 
-The **ID** (External ID) is an unique identifier for the line item.
-Feel free to use the one of your previous software to ease the transition to Odoo.
+The :guilabel:`ID` (External ID) is an unique identifier for the line item. Feel free to use one
+from previous software to facilitate the transition to Odoo.
 
 Setting an ID is not mandatory when importing but it helps in many cases:
 
-* Update imports: you can import the same file several times without creating duplicates;
-* Import relation fields (see here below).
+* Update imports: import the same file several times without creating duplicates;
+* Import relation fields (see below).
```

Comment:
instead of *, I think this would be better utilized as an admonition

Reply:
Changed to bullet points, removed asterisk